java/demo/src/com/google/phonenumbers/PhoneNumberParserServlet.java

```diff
@@ -322,6 +324,7 @@ private StringBuilder getOutputForSingleNumber(
     } catch (NumberParseException e) {
       output.append(e.toString());
```

Comment:
This was by design. It was intended to display the input plus the error.

Before:
![before](https://cloud.githubusercontent.com/assets/9248532/12053614/337f0c8a-af1b-11e5-9e8a-b59cd9c43607.png)

After:
![after](https://cloud.githubusercontent.com/assets/9248532/12053617/3b4343dc-af1b-11e5-9853-929659aecb0d.png)

Killing the request would remove the information about the error which is currently displayed.